app/views/users/index.html.erb

```diff
@@ -0,0 +1,7 @@
+<section>
+<ul>
+<% @user.each do |users| %>
+<li> <%= render 'user',users:users %> </li>
```

Comment:
- [ ] According to the project requirement when we click on each, it should take us to their show page. Unfortunately, when I clicked in I could not go to the page I had to type the URLs. Please fix it and also make sure that your design matches with the [wireframe](https://github.com/microverseinc/curriculum-rails/raw/main/blog-app/images/blog_user_page.png) as require for this project project :+1: